proto/rollup/v1/msg.proto

```diff
@@ -22,15 +22,16 @@ message ApplyL1TxsRequest {
 
 message ApplyL1TxsResponse {}
 
-// MsgInitiateWithdrawal defines a message for all L2 withdrawal txs
+// TODO: add validations in the x/rollup module to ensure that the target address and gas limit are valid before burning the users ETH
```

Comment:
I created a subtask for this on the withdrawals task. We should validate the target address and gas limit here and ensure that an invalid withdrawals hash can't be submitted to the `L1ToL2MessagePasser` contract.